website/javascript/templates/GameFeed.vue

```diff
@@ -0,0 +1,248 @@
+                let next_game = this.incoming_games[this.incoming_games.length - 1]
+                let next_time = moment(next_game.time_played)
+                let cur_time = moment(this.display_games[0].time_played)
+                let delay = Math.min(4000, next_time - cur_time)
+                setTimeout(this.display_next, delay)
+              }
+
+              this.fetch_tries = 0
+              resolve(data)
+            } else {
+              // If we don't get any games try again with a backoff from
+              // 10 seconds up to 5.5 minutes.
+              this.fetch_tries = Math.min(5, this.fetch_tries)
+              let wait_time = 10
+              wait_time += (Math.random() * Math.pow(2, this.fetch_tries)) * 10
+              this.fetch_tries += 1
+              setTimeout(this.fetch, wait_time * 1000)
+              console.log("No games, waiting " + wait_time)
+              reject("No games found")
+            }
+          })
+      });
+    },
+    getFormattedDateForGames: function (date, return_value_not_valid) {
+      var cdate = moment(date)
+      if (cdate.isValid()) {
+        var dateFormat = require('dateformat')
+        return dateFormat(date, 'dd/mm/yy HH:MM:ss')
+      } else {
+        return return_value_not_valid
+      }
+    },
+    display_next: function() {
+      if (this.incoming_games.length) {
+        this.display_games.unshift(this.incoming_games.pop())
+        if (this.display_games.length > 50) this.display_games.pop()
+      }
+      let cur_time = moment(this.display_games[0].time_played)
+      let since_update = moment() - this.last_received
+      if (this.most_recent - cur_time < 30000 && since_update > 20000 &&
+          this.fetch_tries == 0) {
+        this.fetch_tries += 1
+        this.fetch()
+      }
+      if (this.incoming_games.length) {
+        let next_game = this.incoming_games[this.incoming_games.length - 1]
+        let till_next = moment(next_game.time_played) - cur_time
+        let qtime = since_update + this.most_recent - cur_time
+        let speed = qtime / 60000
+        let delay = Math.max(500, till_next / speed)
+        setTimeout(this.display_next, delay)
+      }
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+    .table-leader {
+        table-layout: fixed;
+        white-space: nowrap;
+    }
+    .watch {
+        width: 17rem;
+    }
+    .destroyed {
+        width: 15rem;
+    }
+    .map-size {
+        width: 12rem;
+    }
+    .turns {
+        width: 7rem;
+    }
+    .table-leader .td-wrapper {
+        height: 2.5rem;
+        max-height: 3rem;
+        overflow: hidden;
+        position: relative;
+    }
+    .table-leader .td-wrapper > div {
+        white-space: nowrap;
+        width: 100%;
+        position: absolute;
+        bottom: 0;
+    }
+    .game-participant {
+        img {
+            height: 20px;
+            width: 20px;
+        }
+        .username {
+            font-size: 1rem;
```

Comment:
At least for me, this makes the font size nearly unreadably tiny, and doesn't seem necessary. For mobile, it doesn't fit all that well anyways and is even tinier - maybe we could save some space by shortening the date/time format, if that's the concern.

Reply:
This wasn't even so much about space as wanting to de-emphasize the usernames, the default 1.6rem feels really heavy. But yeah I probably overdid it, I'll bump it back up some.